src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+document.querySelectorAll('.tree > li').forEach((node) => {
+  wrapTextInSpan(node);
+});
+
+function wrapTextInSpan(node) {
+  const span = document.createElement('span');
+
+  span.textContent = node.childNodes[0].textContent.trim();
```

Comment:
Consider adding a check to ensure `node.childNodes[0]` exists before accessing its `textContent`. This will prevent potential errors if the node does not have any child nodes.